src/main/java/com/library/model/mapper/BookMapper.java

```diff
@@ -1,20 +1,30 @@
 package com.library.model.mapper;
 
 import com.library.model.dto.BookDto;
```

Comment:
In addition to comment about new one Book bean:

This DTO is DTO for which entity bean?

Reply:
Entity bean is one (for postgres db)

Reply:
ok. my mistake in terms.

This DTO is DTO for entity or for document? How another developer should understand it from name and package?

Reply:
Entity, documents and their DTO were renamed in commit